### PR TITLE
Add main menu

### LINKED ISSUE #13: Main menu
Make a main menu that users can select the option.

```
=== Personal Finance Tracker ===
0. Import a CSV File
1. View All Transactions
2. View Transactions by Date Range
3. Add a Transaction
4. Edit a Transaction
5. Delete a Transaction
6. Analyze Spending by Category
7. Calculate Average Monthly Spending
8. Show Top Spending Category
9. Visualize Monthly Spending Trend
10. Save Transactions to CSV
11. Exit
Choose an option (1-11):
```

### PR DESCRIPTION
**Pull request by shigeokitamura**
close #13 

Add main menu mock

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5,8 +5,41 @@
...
+            "Show Top Spending Category",
+            "Visualize Monthly Spending Trend",
+            "Save Transactions to CSV",
+            "Exit",
+        ]
+
+        for idx, menu in enumerate(menues):
+            print(f"{idx}. {menu}")
+
+        option = input("Choose an option (1-11): ")
```
Comment (krlove98) [inline main.py]: It should be 0 to 11

Comment (shigeokitamura) [reply main.py]: Changed to `0-11`

File: main.py [hunk 2]
```diff
@@ -5,8 +5,41 @@
...
+            "Exit",
+        ]
+
+        for idx, menu in enumerate(menues):
+            print(f"{idx}. {menu}")
+
+        option = input("Choose an option (0-11): ")
+
+        match option:
+            case "1":
```
Comment (Aspyn25) [inline main.py]: hey~ can you fix the '1' to 0? thanks

Comment (shigeokitamura) [reply main.py]: Changed to "0", thanks

**Review by Aspyn25 — APPROVED** [review]

**Review by krlove98 — APPROVED** [review]

**Pull request merged**